lib/mp4/MP4TagMapper.ts

```diff
@@ -7,6 +7,7 @@ import {CommonTagMapper} from '../common/GenericTagMapper';
 const mp4TagMap: INativeTagMap = {
   '©nam': 'title',
   '©ART': 'artist',
+  '©art': 'artist',
```

Comment:
Should the tag mapping be case insensitive?

Reply:
You're right, it's a better idea, I made it now case insensitive